app/decorators/case/sar/offender_decorator.rb

```diff
@@ -22,8 +22,14 @@ def request_methods_for_display
   end
 
   def rejected_reasons_descriptions
+    other_reason_text = if other_rejected_reason.present?
+                          ": #{other_rejected_reason}"
+                        else
+                          ""
+                        end
+
     rejected_reasons.map { |reason|
       Case::SAR::Offender::REJECTED_REASONS[reason]
-    }.append(other_rejected_reason).compact.join("<br>")
+    }.compact.join("<br>") + other_reason_text
```

Comment:
This relies on OTHER being the last option in the REJECTED_REASONS hash. This is true now, but could change in theory. 
It might then be better to append the other reason within the map when it is the relevant object.